README.md

```diff
@@ -19,18 +19,36 @@ If you are not using CocoaPods, you can integrate the Placed.framework and Place
 
 ## Location Permission
 
-This SDK requires Apple's background location permission. Under the "Required background modes" key in your app’s main plist file you should make sure to add:
-
-    App registers for location updates
-
-In addition, for iOS 11, you must provide all of the following location usage descriptions in your app's plist:
-
-    NSLocationAlwaysAndWhenInUseUsageDescription
-    NSLocationWhenInUseUsageDescription
-    NSLocationAlwaysUsageDescription
-    NSLocationUsageDescription
-
-**These permissions are very important to the Placed SDK. If your app does not currently use background location permission, please contact a Placed representative.**
+As outlined in Section 5 of the [Placed Affiliate Agreement](https://affiliate.placed.com/placed-affiliate-agreement/), you must satisfy two requirements prior to registering a user with the Placed SDK:
+
+1. *Gather Express Consent for User Data Collection via Opt-in Dialog*  
+In addition having a legally compliant privacy policy describing Placed’s collection of location and device information, you must include a discrete opt-in dialog which gathers express consent for data collection. This dialog appears prior to the fine location permission prompt, and includes:  
+    - The language: “*Aggregated device data, including location, is measured for the purposes of market research by Placed, Inc.*”
+    - Links to the Placed [Terms of Service](https://www.placed.com/terms-of-service) and [Privacy Policy](https://www.placed.com/privacy-policy)
+    - Buttons to “Accept” or “Cancel” 
+
+
+1. *Prompt for Background Location Permission*  
+Under the “Required background modes” key in your app’s main plist file, you will need to add:  
+    - **App registers for location updates**   
+    - (for iOS 11) **NSLocationAlwaysAndWhenInUseUsageDescription**
+    - (for iOS 11) **NSLocationWhenInUseUsageDescription**  
+    - (for iOS 11) **NSLocationAlwaysUsageDescription**  
+    - (for iOS 11) **NSLocationUsageDescription**  
+
+For an example of the opt-in dialog and location permission prompt, please refer to the sample app. We have also provided a gallery for inspiration on how you can better integrate the opt-in experience into your app.
+
+<section>
+	<ul>
+    <img src="./Images/aff-iOS-optin-dialogue1.jpg" width=30%>
+    <img src="./Images/aff-iOS-optin-dialogue2.jpg" width=30%>
+    <img src="./Images/aff-iOS-optin-full1.jpg" width=30%>
+    <img src="./Images/aff-iOS-optin-full2.jpg" width=30%>
+    <img src="./Images/aff-iOS-optin-full3.jpg" width=30%>
+  </ul>
+</section>
```

Comment:
Does this gallery work in github? I don't see it when I view this file.

Reply:
Well... great, it works only in chrome (at least for me)

Reply:
I can't see it looking here: https://github.com/sewichi/ios-placed-sdk/blob/d821dc3be115927a2b0c0f76de2c04ea5057b5e7/README.md

Reply:
Kyle just told me that he actually wanted to have images as a link, so this change is not applicable anymore.